vectordatasource/transform.py

```diff
@@ -9646,3 +9669,37 @@ def admin_level_alternate_viewpoint(shape, props, fid, zoom):
                 props['kind:' + viewpoint] = _ADMIN_LEVEL_TO_KIND.get(admin_level, None)
 
     return shape, props, fid
+
+
+def unpack_places_disputes(shape, props, fid, zoom):
+    """
+    turns disputed places into 'unrecognized' for that viewpoint
+    """
+    disputed_by = props.pop('disputed_by', '')
+    disputants = _list_of_countries(disputed_by)
+
+    for disputant in disputants:
+        props['kind:' + disputant] = 'unrecognized'
```

Comment:
We need to be consistent with what we're doing in the boundaries layer, and say `unrecognized_` postfixed with the original kind, please.

So:

- unrecognized_country
- unrecognized_region
- unrecognized_county
- unrecognized_locality

Reply:
Same logic as you have up in `unpack_viewpoint_claims()`.

Reply:
maybe a dumb question but... what's the point of having any further detail on an item if it's unrecognized by that viewpoint?  `unrecognized` just turns off the feature, right?  Shouldn't we just make everything `unrecognized` and remove all the suffixes?